ports/beman-exemplar/portfile.cmake

```
@@ -0,0 +1,29 @@
+# vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO bemanproject/exemplar
+    REF "${VERSION}"
+    SHA512 492dea71e64f3ee08e502c8eda33697d30f9eb9c04f32b2e1f4ffad28a605448e47dd6242356033ce06fcc972d0816348d44cfb2bd0dead12dffffe3d9ab8e16
+    HEAD_REF main
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(
+    PACKAGE_NAME "beman-exemplar"
```

```suggestion
    PACKAGE_NAME "beman.exemplar"
```
for `beman.exemplar-config.cmake` to be found

---

Thanks!